requirements.txt

```diff
@@ -1,3 +1,6 @@
 python-telegram-bot
 requests
-beautifulsoup4
\ No newline at end of file
+beautifulsoup4
+json
```

Comment:
json and os aren't dependencies that need to be installed, it's part of the standard library. In fact it will lead to an error if you try to install it.